tests/components/weather/test_weather.py

```diff
@@ -56,12 +57,20 @@ def test_attributes(self):
         assert data.get(ATTR_WEATHER_ATTRIBUTION) == "Powered by Home Assistant"
         assert data.get(ATTR_FORECAST)[0].get(ATTR_FORECAST_CONDITION) == "rainy"
         assert data.get(ATTR_FORECAST)[0].get(ATTR_FORECAST_PRECIPITATION) == 1
+        assert (
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5710832)